lib/src/boilerplate_suggestors/advanced_props_and_state_class_migrator.dart

```diff
@@ -30,14 +30,24 @@ class AdvancedPropsAndStateClassMigrator extends GeneralizingAstVisitor
     with AstVisitingSuggestorMixin
     implements Suggestor {
   final ClassToMixinConverter converter;
+  final SemverHelper semverHelper;
 
-  AdvancedPropsAndStateClassMigrator(this.converter);
+  AdvancedPropsAndStateClassMigrator(this.converter, this.semverHelper);
 
   @override
   void visitClassDeclaration(ClassDeclaration node) {
     super.visitClassDeclaration(node);
 
-    if (!shouldMigrateAdvancedPropsAndStateClass(node)) return;
+    if (shouldAddPublicExportLocationsAdvancedClassComment(
+        node, semverHelper)) {
+      addPublicExportLocationsComment(
+          node, sourceFile, semverHelper, yieldPatch);
+    }
+
+    if (!shouldMigrateAdvancedPropsAndStateClass(node, semverHelper)) return;
+
+    removePublicExportLocationsComment(
```

Comment:
Good question... 

I added quite a bit of logic in https://github.com/Workiva/over_react_codemod/pull/82 that associates "FIX ME" comments with migration decisions.  IMO - the stuff in this PR involving the adding/removing of comments could be removed for now - and then we could circle back to make the comment added using the logic in #82 have more informative content _(like where the member is exported from, etc)_

Reply:
I remove the comments here because at this point, the class is going to be updated so any existing comments explaining why the class wasn't updated should be removed. `removePublicExportLocationsComment` only yields a patch if the comment exists.